pubsub/cloud-client/src/main/java/com/example/pubsub/SubscriberExample.java

```diff
@@ -28,18 +28,29 @@
 import java.util.concurrent.LinkedBlockingDeque;
 
 public class SubscriberExample {
+  static class MessageAndConsumer {
+    PubsubMessage message;
+    AckReplyConsumer consumer;
+
+    MessageAndConsumer(PubsubMessage message, AckReplyConsumer consumer) {
+      this.message = message;
```

Comment:
This should probably use getters?

Reply:
Getters feels a little like overkill, doesn't it? This is just a POD object with these two fields. I don't think defining getters and using them is really going to increase readability here.

Reply:
It's less about readability, and more about promoting best practices. In Java getters and setters are standard and idiomatic, and we want to make sure to reinforce those practices to our users (as some are likely beginners and may not be able to tell when to cut corners and when not)